src/serializers/index.ts

```diff
@@ -32,8 +32,8 @@ const req = (request: Request) =>
         method: request.method,
         url: request.url,
         headers: request.headers,
-        remoteAddress: request?.connection?.remoteAddress,
-        remotePort: request?.connection?.remotePort,
+        remoteAddress: request?.socket?.remoteAddress,
```

Comment:
https://github.com/pinojs/pino-std-serializers#exportsreqrequest

Looks like Pino already serializes these in their standard library. Should we just remove our ones?

Reply:
I think we can switch to the standard serializers if

1. They are supersets of ours
2. The additional fields are useful and we ensure that we have appropriate redaction configured

Reply:
ah ours has a slightly different behaviour, I'll leave it for now